runtime/turing/src/lib.rs

```diff
@@ -1016,11 +1017,13 @@ where
 
 impl pallet_automation_time::Config for Runtime {
 	type Event = Event;
-	type MaxTasksPerSlot = ConstU32<256>;
 	type MaxExecutionTimes = ConstU32<24>;
 	type MaxScheduleSeconds = MaxScheduleSeconds;
 	type MaxBlockWeight = MaxBlockWeight;
 	type MaxWeightPercentage = MaxWeightPercentage;
+	// Roughly .125% of parachain block weight per hour
+	// ≈ 500_000_000_000 (MaxBlockWeight) * 300 (Blocks/Hour) * .00125
+	type MaxWeightPerSlot = ConstU128<150_000_000_000>;
```

Comment:
Roughly, 500 native transfer tasks per hour. Theoretically much lower than our actual capacity however I didn't want to couple this change with a significantly higher capacity (currently 256).